basicswap/bin/prepare.py

```diff
@@ -359,6 +374,36 @@ def setTorrcVars():
     #   'daemon-ssl-allow-any-cert=1', moved to startup flag
 ]
 
+salviumd_proxy_config = [
+    f"proxy={TOR_PROXY_HOST}:{TOR_PROXY_PORT}",
+    "proxy-allow-dns-leaks=0",
+    "no-igd=1",  # Disable UPnP port mapping
+    "hide-my-port=1",  # Don't share the p2p port
+    "p2p-bind-ip=127.0.0.1",  # Don't broadcast ip
+    "in-peers=0",  # Changes "error" in log to "incoming connections disabled"
+    "out-peers=24",
+    f"tx-proxy=tor,{TOR_PROXY_HOST}:{TOR_PROXY_PORT},disable_noise,16",  # Outgoing tx relay to onion
+]
+
```

Comment:
Does salvium have onion seed nodes? And a healthy onion network? (Has peers?)

If yes, this can be deleted.

If no, you'll have to edit this to remove tx-proxy